tests/session/test_env_select.py

```diff
@@ -15,6 +15,57 @@
 
 CURRENT_PY_ENV = f"py{sys.version_info[0]}{sys.version_info[1]}"  # e.g. py310
 
+####################################################################
```

Comment:
I'm -1 on adding these markers 🤔

Reply:
Well, I'm focused here on making this code quicker to understand for people not as familiar with it as I guess you are. If you insist, I'll remove them, but having this when I started working on this file would have saved me some time and made my life easier.

Reply:
I insist 😊 If we need some extra information, let's add it to the docstring of the test IMHO.

Reply:
Ok. Please next time say if you have to have something changed, rather than just saying you're "-1". Those lines are removed.

Putting info in the docstrings doesn't serve the purpose. The point of grouping things is that you can quickly see the groupings and understand something common to all the elements of the group without having to look at each individual element.

Reply:
A better solution in pytest to achieve that is https://stackoverflow.com/questions/50016862/grouping-tests-in-pytest-classes-vs-plain-functions, if that's your intent.

Reply:
That, again, is (IMHO, of course) extra complexity and extra verbosity to do something a simple comment can do. You'll notice that neither of the "two compelling use cases" mentioned in the top-voted answer is related to my purpose here.

That said, thanks for pointing this out: though it doesn't solve the problem I'm trying to solve here, the ability to parametrize a group of tests that particular way is interesting. (I am not sure if it's any better than just assigning the parameters to a variable and re-using that, but it's a technique worth considering next time I run into a need for something along those lines.)

Reply:
I'd argue that comments get stale easily, but adding a new indentation or moving it to a different file makes the intent much more explicit and hard to miss. IMHO, comments should be only used within methods to explain why we're doing something (not the how, for that, the code should be self-documenting).